tests/unit/processes/ingest/test_chicago_isac_process.py

```diff
@@ -14,91 +14,91 @@ def teardown_class(cls):
 
-    def test_processChicagoISACRecord_error(self, mocker):
+    def test_process_chicago_isac_record_error(self, mocker):
 
-        mockMapper = mocker.patch('processes.ingest.chicago_isac.ChicagoISACMapping')
-        mockMapper.side_effect = MappingError('testError')
+        mock_mapper = mocker.patch('processes.ingest.chicago_isac.ChicagoISACMapping')
+        mock_mapper.side_effect = MappingError('testError')
 
         assert pytest.raises(MappingError)
 
-    def test_storePDFManifest(self, testProcess, mocker):
-        mockRecord = mocker.MagicMock(identifiers=['1|isac'])
-        mockRecord.has_part = [
-            ['1|testURI|isac|application/pdf|{}',
-            '2|testURIOther|isac|application/pdf|{}'],
+    def test_store_pdf_manifest(self, test_process, mocker):
+        mock_record = mocker.MagicMock(identifiers=['1|isac'])
+        mock_record.has_part = [
+            ['1|test_url|isac|application/pdf|{}',
+            '2|test_url_other|isac|application/pdf|{}'],
         ]
 
-        mockGenerateMan = mocker.patch.object(ChicagoISACProcess, 'generateManifest')
-        mockGenerateMan.return_value = 'testJSON'
-        mockCreateMan = mocker.patch.object(ChicagoISACProcess, 'createManifestInS3')
+        mock_generate_man = mocker.patch.object(ChicagoISACProcess, 'generate_manifest')
+        mock_generate_man.return_value = 'test_json'
+        mock_create_man = mocker.patch.object(ChicagoISACProcess, 'createManifestInS3')
 
-        testProcess.storePDFManifest(mockRecord)
+        test_process.store_pdf_manifest(mock_record)
 
-        testManifestURI = 'https://test_aws_bucket.s3.amazonaws.com/manifests/isac/1.json'
-        assert mockRecord.has_part[0] == '1|{}|isac|application/webpub+json|{{}}'.format(testManifestURI)
+        test_manifest_url = 'https://test_aws_bucket.s3.amazonaws.com/manifests/isac/1.json'
+        assert mock_record.has_part[0] == '1|{}|isac|application/webpub+json|{{}}'.format(test_manifest_url)
 
-        mockGenerateMan.assert_called_once_with(mockRecord, 'testURI', testManifestURI)
-        mockCreateMan.assert_called_once_with('manifests/isac/1.json', 'testJSON')
+        mock_generate_man.assert_called_once_with(mock_record, 'test_url', test_manifest_url)
+        mock_create_man.assert_called_once_with('manifests/isac/1.json', 'test_json')
 
-    def test_createManifestInS3(self, testProcess, mocker):
-        mockPut = mocker.patch.object(ChicagoISACProcess, 'putObjectInBucket')
+    def test_create_manifest_in_S3(self, test_process, mocker):
+        mock_put = mocker.patch.object(ChicagoISACProcess, 'putObjectInBucket')
         
-        testProcess.createManifestInS3('testPath', '{"data": "testJSON"}')
+        test_process.createManifestInS3('test_path', '{"data": "test_json"}')
 
-        mockPut.assert_called_once_with(b'{"data": "testJSON"}', 'testPath', 'test_aws_bucket')
+        mock_put.assert_called_once_with(b'{"data": "test_json"}', 'test_path', 'test_aws_bucket')
```

Comment:
I think we can remove this test as this method is no longer part of the CHicagoISACProcess class

Reply:
That's true. I'll remove it.